src/tribol/interface/tribol.cpp

```diff
@@ -1008,6 +1028,7 @@ void finalize( )
    {
       csManager.clearAllCouplings();
    }
+   axom::slic::finalize();
```

Comment:
I'm not sure that we want to call ``slic::finalize()`` here either. I think it should be the host-code's responsibility (i.e. at a higher level).

We might want to explicitly use ``slic::SimpleLogger`` in the tribol tests and examples, which will initialize and finalize slic as needed -- https://github.com/LLNL/axom/blob/develop/src/axom/slic/core/SimpleLogger.hpp

... or perhaps create a simple extension in tribol (or axom) that works with MPI

Reply:
I removed this. And `slic::SimpleLogger` is currently being used in tribol tests and examples.